pages/docs/tutorials/streetlights.md

```diff
@@ -1,244 +1,263 @@
-</CodeBlock>
-
-### 5. And voilà! List all files in directory and check that Node.js application is generated:
-<CodeBlock language="bash">
-{`cd output && ls`}
-</CodeBlock>
-
-# Running your code
-
-### 1. Install dependencies of newly generated application:
-<CodeBlock language="bash">
-{`npm install`}
-</CodeBlock>
-
-### 2. Start the application:
-<CodeBlock language="bash">
-{`npm start`}
-</CodeBlock>
-
-### 3. In another terminal install the MQTT.js library:
-<CodeBlock language="bash">
-{`npm install mqtt -g`}
-</CodeBlock>
-
-### 4. Send a correct message to your application:
-<CodeBlock language="bash">
-{`mqtt pub -t 'light/measured' -h 'test.mosquitto.org' -m '{"id": 1, "lumens": 3, "sentAt": "2017-06-07T12:34:32.000Z"}'`}
-</CodeBlock>
-
-### 5. Send an incorrect message to your application:
-<CodeBlock language="bash">
-{`mqtt pub -t 'light/measured' -h 'test.mosquitto.org' -m '{"id": 1, "lumens": "3", "sentAt": "2017-06-07T12:34:32.000Z"}'`}
-</CodeBlock>
-
-### 6. Go back to the previous terminal and check if your application logged the streetlight condition you just sent, with errors related to the invalid message.
-
-# Conclusion
-
-You've learned how to create an AsyncAPI description file and how to generate code from it. Today's sample code is only a bootstrap; you'll need to add your own business logic into it. Take some time to play with it. There are still lots of things to be covered, but the intent of this tutorial is to make it simple for you to get an idea of the potential.
-
-We would love to see what you create with AsyncAPI! As an open-source project, we're open to proposals, questions, suggestions, and contributions. If you don't feel in the mood to contribute but you're using AsyncAPI, just raise your hand by [creating an issue in our Github repo](https://github.com/asyncapi/asyncapi/issues/new) or [join our Slack channel](https://www.asyncapi.com/slack-invite/). Don't be shy.
-
----
-
-<DocsButton
-  suggestions={[
-    {
-      href: '/docs/tutorials',
-      title: 'Tutorials - Overview',
-      type:'back',
-    },
-    {
-      href: '/docs/tutorials/streetlights-interactive',
-      title: 'Streetlights - Interactive (Alpha)',
-      type:'next',
-    }
-  ]}
-/>
\ No newline at end of file
+---
+title: Streetlights
+description: In this tutorial, you'll get started with code samples from an IoT real-world use case. Let's pretend you have a company called Smarty Lighting that engineers smart-city lighting systems.
+---
+
+# Introduction
+
+In this tutorial, you get started with actual code and a sample real-world use case. Let's pretend you have a company called Smarty Lighting and you install smart-city streetlight lighting systems. This is a use case of Internet of things (IoT).
+
+We are going to use Node.js and Mosquitto as the message broker to build a Smartylighting Streetlights API which allows you to manage city lights remotely.

```

Comment:
```suggestion
You're going to use Node.js and Mosquitto as the message broker to build a Smartylighting Streetlights API, which allows you to manage city lights remotely.

```

Something about these first docs is that they lack style guide consistency in a few items. In this case, I want you to notice that this tutorial is not consistent with the use of `we` and `you`. There are 17 places where `you` was used, as opposed to 7 times that `we` was used.

For our docs, I want us to stick to using `you` when addressing our readers with info/instructions on how to accomplish their goals. 

👉🏽 Let's go ahead and edit all mentions of `we` and replace them for `you`.